packages/devtools_app/lib/src/screens/memory/panes/allocation_profile/allocation_profile_table_view_controller.dart

```diff
@@ -66,7 +67,15 @@ class AllocationProfileTableViewController extends DisposableController
   /// Converts the current [AllocationProfile] to CSV format and downloads it.
   ///
   /// The returned string is the name of the downloaded CSV file.
-  String downloadMemoryTableCsv(AllocationProfile profile) {
+  Future<String?> downloadMemoryTableCsv(AllocationProfile profile) async {
+    final path = await getSavePath(
```

Comment:
can you file an issue to upstream this user-specified location option into export controller directly? That way other screens can leverage this functionality.